### DISCUSSION TITLE
How does the toggle feature work? Can I achieve the same thing using the command line?

### DISCUSSION THREAD

**Discussion by morristai** [intellij-rust/intellij-rust#10355] [Q&A] [answered]
In `Cargo.toml` file, how does these toggles work? Is this a feature built in intellij-rust or cargo? Can I achieve the same thing using the command line? Thanks

![image](https://user-images.githubusercontent.com/10728152/232276575-d5a0a5cc-6fda-48a3-8860-5777610c823a.png)

**Comment by Undin** [comment, marked as answer]
It's intellij-rust feature that allows you to configure feature-set for code analysis.
From Cargo point of view, you can't enable/disable any of these features in a library because a library only provides its set of features for clients, and clients choose what features they need

**Reply by morristai** [reply]
thanks @Undin
